### PR TITLE
Recipe builder

### PR DESCRIPTION
**Pull request by fonuxxine**
Add Recipe Buider Formatter
Update Docstring

### CODE REVIEW

**Review by EricKarpovits — APPROVED** [review]
This looks great @fonuxxine!

File: src/main/java/presenters/RecipeBuilderFormatter.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package presenters;
+
+import useCases.RecipeBuilderResponseModel;
+
+public class RecipeBuilderFormatter implements RecipeBuilderPresenter{
```
Comment (EricKarpovits) [inline src/main/java/presenters/RecipeBuilderFormatter.java:5]: Add some docstrings to this class please

File: src/main/java/useCases/RecipeBuilderResponseModel.java [hunk 2]
```diff
@@ -19,17 +19,20 @@ public RecipeBuilderResponseModel(RecipeFactory recipeFactory) {
     public ArrayList<Recipe> createRecipe(String apiOutput){
         RecipeFactory factory = new CommonRecipeFactory();
         ArrayList<Recipe> formattedRecipes = new ArrayList<Recipe>();
```
Comment (EricKarpovits) [inline src/main/java/useCases/RecipeBuilderResponseModel.java:21]: Add some comments to this file please

**Review by rickychen175 — APPROVED** [review]
Looks good. I think it would be helpful to document the difference in behavior when an exception is encountered.

**Review by RedFoxity — APPROVED** [review]
Everything looks like its in order, good job.

**Pull request merged**